podcastfy/content_parser/content_extractor.py

```diff
@@ -62,15 +66,17 @@ def extract_content(self, source: str) -> str:
 			ValueError: If the source type is unsupported.
 		"""
 		try:
-			if self.is_url(source):
```

Comment:
it's better to have a dedicated method for that as self.is_url, curious the motivation to remove